CHANGELOG.md

```diff
@@ -2,7 +2,13 @@
 
 Inspired by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-Note: this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+**Note:** this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [2.15.0] - 2023-03-06
```

Comment:
Nit: we usually put "Unreleased" here and then put the date in later, when we release it. Not that it matters much...but it helps us remember that we need to fill the date in.